plugins/sqlfluff-plugin-example/src/example/rules.py

```diff
@@ -47,7 +47,7 @@ class Rule_Example_L001(BaseRule):
     | **Anti-pattern**
     | Using ORDER BY one some forbidden columns.
 
-    .. code-block::
+    .. code-block:: sql
```

Comment:
I am not sure could we add unittest for plugin, So I just add syntax hint for it

Reply:
Yes that's all I meant. Resolving.